packages/api/internal/handlers/sandboxes_list.go

```diff
@@ -64,23 +73,11 @@ func parseCursor(cursor string) (time.Time, string, error) {
 	return cursorTime, parts[1], nil
 }
 
-func (a *APIStore) getRunningSandboxes(runningSandboxes []*instance.InstanceInfo, metadataFilter *map[string]string, cursorTime time.Time, cursorID string, limit *int32) ([]PaginatedSandbox, error) {
+func transformToSandboxAPI(runningSandboxes []*instance.InstanceInfo) []PaginatedSandbox {
```

Comment:
I got confused by this, why is this necessary?

Reply:
It isn't strictly necessary, but I wanted to make the function smaller, so they don't do everything. This way we can reuse some of the functions for multiple endpoints without any weird logic for making some parameters optional, but then actually requiring them later

Reply:
Feel free to propose better name for this function though, I struggled with that :D